packages/rum-recorder/src/domain/rrweb-snapshot/snapshot.ts

```diff
@@ -0,0 +1,589 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/prefer-regexp-exec */
```

Comment:
I wasn't sure on this.

Example: https://github.com/DataDog/browser-sdk/pull/700/files#diff-1c7cef52f64a686ba523420f60390086b7a16d8f13b51ac8f0a3e79d90be827eR349

While replacing a `'foo'.match()` to `/foo/.exec()` makes sense, here we're talking about creating a new regexp from the return value of `lowerIfExists`?

Reply:
I think it simply means to change
```ts
lowerIfExists(sn.attributes.name).match(/^msapplication-tile(image|color)$/)
```
to
```ts
/^msapplication-tile(image|color)$/.exec(lowerIfExists(sn.attributes.name))
```
also we could take the opportunity to change `exec` to `test` when the result is used as a boolean

```ts
/^msapplication-tile(image|color)$/.test(lowerIfExists(sn.attributes.name))
```

Reply:
ah, I didn't get the string & regexp were inverted between match and exec. I'll make the change